static/js/src/navigation.js

```diff
@@ -115,6 +115,22 @@ function handleDropdownClick(clickedDropdown) {
   setTabindex(clickedDropdown.querySelector("ul.p-navigation__dropdown"));
 }
 
+function updateUrlHash(id, open) {
+  if (id !== null && id !== undefined && open) {
```

Comment:
Does it not work if ?
```
if (id && open) {
```

Reply:
I did this as I wanted to allow the url hash to be the other falsey values, but I don't think it is actually needed. So will go with your way.